mkdocs.yml

```diff
@@ -12,9 +12,6 @@ docs_dir: docs
 
 nav:
   - ehrQL: index.md
-  - Introduction:
-    - Guidance for existing cohort-extractor users: introduction/guidance-for-existing-cohort-extractor-users.md
-    - Introduction to ehrQL concepts: introduction/introduction-to-concepts.md
   - Tutorial:
     - Tutorial: tutorial/index.md
```

Comment:
I would use _tutorials_ here because we have more than one. We also use plural in the 'How-to guides' subsection